### ISSUE TITLE
Installation..

### ISSUE THREAD

**Issue by irrelevantdotcom** [inveniosoftware/troubleshooting#28]
Hi.

I've come to Invenio as it appears to fulfil my needs, based on the demo system and other users.  I have a mountain of potentially historically relevant documents to collate and catalogue, following the death of my father and my taking over responsibility for his records and archives, and after much searching, it seems to fit my need for a "comprehensive but straightforward" document management system.

Obviously I start on http://invenio-software.org/gettingstarted and try the installation steps.  As I already have docker running on my test server, I follow the 3.0 instructions. Everything goes perfectly and it all fires up.  Except there is almost nothing available in the web interface yet...  I read further and find 3.0 is still very much in dev.  The other option is for 1.3, using vagrant, which I have zero experience with so far. And what happened to v2 ?

So I then find http://invenio.readthedocs.io/en/stable/developers/docker.html 
which (also) starts with
`docker-compose -f docker-compose-dev.yml build 
docker-compose -f docker-compose-dev.yml up`
(the line break is wrong on that page, BTW)
I download a copy of maint2.1 version from github, and it builds, but fails with loads of errors on the "up" command.  The first one seems to be `pkg_resources.ContextualVersionConflict: (Pillow 2.9.0 (/usr/local/lib/python2.7/site-packages), Requirement.parse('pillow>=3.4'), set(['Flask-IIIF']))` but there are lots more...

So ... is 2.1 the latest "working" version?  And what do I really need to do to install it?   Although my history is in development, so I don't mind getting dirty with code, I'm really just trying to install this to use..

**Comment by lnielsen** [comment]
Currently, v1 is the latest stable release, but it's quite a legacy system. V2 is a hybrid release (legacy vs new) that we do not recommend using as it proved to be quite unstable in a production environment. V3 is not yet released (we're aiming at April for the final release).

**Comment by MarcoGiorgi** [comment]
Is version 3 ready to use in production?

**Comment by lnielsen** [comment]
Yes - see https://inveniosoftware.org/blog/invenio-v300-released/
